src/appleseed.studio/mainwindow/rendering/renderwidget.h

```diff
@@ -50,10 +51,15 @@ namespace OCIO = OCIO_NAMESPACE;
 // Standard headers.
 #include <cstddef>
 #include <memory>
+#include <string>
 
 // Forward declarations.
 namespace foundation    { class CanvasProperties; }
 namespace renderer      { class Frame; }
+namespace renderer      { class Project; }
```

Comment:
You no longer need `Project` anywhere in the render widget, right?

Reply:
It s still here because I am still thinking for a way to remove the assign_material from renderingmanager and I will need project to implement it in RenderWidget

Reply:
We don't want the code for material assignment in `RenderWidget` either. Render widget is (mostly) responsible for show render results, and that should be pretty much it. We don't want to add anything more to it. It's the Single Responsibility Principle: https://www.google.com/search?q=Single+Responsability+Principle&oq=Single+Responsability+Principle